converter/stitching/session/session.go

```diff
@@ -253,3 +253,37 @@ func (s *Aggregate) ToRITAConn(conn *parsetypes.Conn, localFunc func(string) boo
 		conn.RespIPBytes = int64(s.OctetTotalCountAB)
 	}
 }
+
+func FlowStartMilliseconds(f *Aggregate) int64 {
```

Comment:
Let's change this to `func (f *Aggregate) FlowStartMilliseconds() int64`. That will attach the function to the type as a method. Then you can write `sessAgg.FlowStartMilliseconds()` instead of `FlowStartMilliseconds(&sessAgg)`